tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "preserve"
+    "jsx": "react",
+    "noFallthroughCasesInSwitch": true
```

Comment:
What do these 2 options do?

Reply:
![image](https://user-images.githubusercontent.com/11858848/111195869-168da380-8593-11eb-987a-d9af77e5a6fc.png)

Reply:
![image](https://user-images.githubusercontent.com/11858848/111196135-581e4e80-8593-11eb-9d94-9afc9bb0f924.png)

I don't remember why I had to make this change